docs/document-creator/config-file.md

```diff
@@ -263,6 +264,20 @@ By default, if this field is not present, it will create the document with "tt"
 
 _i.e. "document-1.tt"_
 
+### `"fileName"` field
+
+```json
+"fileName": "ebl-<%= ebl-number %>",
+```
+
+_Note: This fileName field is optional. Only add this section in if you want to customise the file name to follow the form fields when uploading data file._
+
+The `"fileName"` field is a string that refers to the file name of the created document. The method uses interpolate delimiters to interpolate properties of data from data file. For further details on the implementation, you can reference : https://www.geeksforgeeks.org/lodash-_-template-method/
```

Comment:
ok, seems ok, but i think referencing the geeks for geeks website is not a good idea. maybe remove the line: For further details ..... 
than ok liao